web/packages/teleterm/src/ui/Search/useSearch.ts

```diff
@@ -114,6 +114,10 @@ export function useFilterSearch() {
               .includes(search.toLocaleLowerCase())
           );
         }
+        // Cluster filter should not be visible if there is only one cluster
+        if (clusters.length === 1) {
+          return [];
+        }
```

Comment:
I suppose if there was zero clusters that would mean they weren't logged in at all so we don't really need to handle that case? in the future if we plan to be able to use this outside of being logged in, maybe we'd check `< 2` instead. not a blocker, just a discussion

Reply:
>I suppose if there was zero clusters that would mean they weren't logged in at all so we don't really need to handle that case? 

Yes. 

> in the future if we plan to be able to use this outside of being logged in, maybe we'd check < 2 instead.

This is already handled, in the next line there is `return clusters.map...` which returns an empty array if an empty array is given.